votes/0639-propose-a-release-note-guide.md

```diff
@@ -0,0 +1,76 @@
+
+**Examples:**
+
+| **Not recommended**                                          | **Clear in type** | **Adequate and clear in meaning** | **User perspective** | Recommended                                                  |
+| ------------------------------------------------------------ | ----------------- | --------------------------------- | -------------------- | ------------------------------------------------------------ |
+| lock_resolver: avoid pessimistic transactions using resolveLocksForWrite | ❌                 | ✅                                 | ❌                    | Fix the issue that committing pessimistic transactions might cause write conflict |
+| retry when meeting stablish conn fails                       | ❌                 | ❌                                 | ❌                    | Fix the issue of unexpected results when TiFlash fails to establish MPP connections |
+| Fix the issue that greatest(datetime) union null returns empty string | ✅                 | ❌                                 | ✅                    | Fix the issue that the query result might be wrong when NULL is in the UNION subquery |
+| copr: make CM Sketch built with the same encoding as what TiDB assumes | ❌                 | ❌                                 | ❌                    | Fix the wrong statistics caused by the format inconsistency of CM Sketch between TiKV and TiDB |
+
+## Improvement
+
+An improvement note means that your PR improves stability or performance of the product, or enhances an existing feature. In addition to describing what your PR has changed, you should also mention how users can benefit from it.
+
+This type of release note consists of two parts: what you have changed + the benefit of your change. This type of release notes often starts with "support", "increase", "improve", "optimize", etc.
+
+**Examples:**
+
+| **Not recommended**                                          | **Clear in type** | **Adequate and clear in meaning** | **User perspective** | Recommended                                                  |
+| ------------------------------------------------------------ | ----------------- | --------------------------------- | -------------------- | ------------------------------------------------------------ |
+| Not use the stale read request's `start_ts` to update `max_ts` to avoid commit request keep retrying | ✅                 | ✅                                 | ✅                    | Avoid excessive commit request retrying by not using the Stale Read request's start_ts to update `max_ts` |
+| Restore many small tables would be faster.                   | ✅                 | ❌                                 | ❌                    | Split and scatter Regions concurrently to improve restore speed |
```

Comment:
I prefer the original description, it's short and doesn't use TiDB terms 🤔

Reply:
Each release note is expected to start with a do-form verb. The original release note "Restore many small tables would be faster." does not accurately and adequately summarize what the PR has changed. "would" looks like a bit uncertain.